Workbooks/Azure Resources/Azure Resource Health/Azure Resource Health.workbook

```diff
@@ -0,0 +1,247 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "value::all"
+        ],
+        "parameters": [
+          {
+            "id": "b09a0fdd-5b98-4304-bb0d-0d2e71b0df9c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscriptions",
+            "type": 6,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::1"
+            ],
+            "typeSettings": {
+              "limitSelectTo": 100,
+              "additionalResourceOptions": [
+                "value::1",
+                "value::all"
+              ],
+              "includeAll": false
+            }
+          },
+          {
+            "id": "bdfc7b21-2dd7-4619-9f9e-7b81b698ad9c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resources",

```

Comment:
not sure how/if this works?
you have it as a resource *type* parameter, but then below the subscriptions parameter is used?  doing it that way would cause the health provider to filter the subscriptions param to resources of the selected type?  so it should never work?

you could change this param to instead query arm for the health provider's supported resource types and show just the types that health works on? let me know if you need details how to do that.  not sure if that api returns "subscription" as an option or not?